src/libraries/Actions.sol

```diff
@@ -10,33 +10,37 @@ library Actions {
     uint256 constant DECREASE_LIQUIDITY = 0x01;
     uint256 constant MINT_POSITION = 0x02;
     uint256 constant BURN_POSITION = 0x03;
+    uint256 constant INCREASE_LIQUIDITY_FROM_DELTAS = 0x04;
+    uint256 constant MINT_POSITION_FROM_DELTAS = 0x05;
+
     // swapping
-    uint256 constant SWAP_EXACT_IN_SINGLE = 0x04;
-    uint256 constant SWAP_EXACT_IN = 0x05;
-    uint256 constant SWAP_EXACT_OUT_SINGLE = 0x06;
-    uint256 constant SWAP_EXACT_OUT = 0x07;
+    uint256 constant SWAP_EXACT_IN_SINGLE = 0x06;
+    uint256 constant SWAP_EXACT_IN = 0x07;
+    uint256 constant SWAP_EXACT_OUT_SINGLE = 0x08;
+    uint256 constant SWAP_EXACT_OUT = 0x09;
     // donate
-    uint256 constant DONATE = 0x08;
+    uint256 constant DONATE = 0x10;
 
     // closing deltas on the pool manager
     // settling
-    uint256 constant SETTLE = 0x09;
-    uint256 constant SETTLE_ALL = 0x10;
-    uint256 constant SETTLE_PAIR = 0x11;
+    uint256 constant SETTLE = 0x11;
+    uint256 constant SETTLE_ALL = 0x12;
+    uint256 constant SETTLE_PAIR = 0x13;
     // taking
-    uint256 constant TAKE = 0x12;
-    uint256 constant TAKE_ALL = 0x13;
-    uint256 constant TAKE_PORTION = 0x14;
-    uint256 constant TAKE_PAIR = 0x15;
+    uint256 constant TAKE = 0x14;
+    uint256 constant TAKE_ALL = 0x15;
+    uint256 constant TAKE_PORTION = 0x16;
+    uint256 constant TAKE_PAIR = 0x17;
+
+    uint256 constant SETTLE_TAKE_PAIR = 0x18;
+    uint256 constant CLOSE_CURRENCY = 0x19;
+    uint256 constant CLEAR_OR_TAKE = 0x20;
```

Comment:
Just curious, why the aversion to hex digits `abcdef`? `0x18 + 0x2 = 0x1a`

Reply:
Just a mistake I think